instrumentation/couchbase/couchbase-3.1/javaagent/build.gradle.kts

```diff
@@ -31,7 +31,7 @@ dependencies {
     ),
   )
 
-  library("com.couchbase.client:java-client:3.1.0")
+  library("com.couchbase.client:java-client:3.1.4")
```

Comment:
could you add a comment here that we are are using `3.1.4` instead of `3.1.0` because of test stability and because M1 requires a server version that does not work with `3.1.0`